src/components/UtilizationBar/UtilizationBar.js

```diff
@@ -0,0 +1,93 @@
+import { ProgressBar } from '../ProgressBar';
+import PropTypes from 'prop-types';
+import { OverlayTrigger } from '../OverlayTrigger';
+import { Tooltip } from '../Tooltip';
+import React from 'react';
+
+const AvailableTooltipFunction = (max, now) => {
+  return <Tooltip id={Math.random()}>Available {max - now} %</Tooltip>;
+};
+
+const UsedTooltipFunction = now => {
+  return <Tooltip id={Math.random()}>Used {now} %</Tooltip>;
+};
+
+const UtilizationBar = props => {
+  const barStyle = () => {
+    if (props.thresholdWarning && props.thresholdError) {
+      let style = 'success';
+      if (props.thresholdWarning <= props.now) {
+        style = 'warning';
+      }
+      if (props.thresholdError <= props.now) {
+        style = 'danger';
+      }
+      return style;
+    } else {
+      return null;
+    }
+  };
+
+const mainDivClasses = mainDivClasses => {
+  if (mainDivClasses) {
+    return mainDivClasses + " progress";
+  } else {
+    return "progress";
+  }
+};
+
+  return (
+    <div className={mainDivClasses(props.mainDivClasses)}>
+      <OverlayTrigger
+        overlay={props.usedTooltipFunction(props.now)}
+        placement="top"
+        trigger={['hover', 'focus']}
+        rootClose={false}
+      >
+        <ProgressBar
+          bsStyle={barStyle()}
+          min={props.min}
+          max={props.max}
+          now={props.now}
+          key={1}
+          isChild
+        />
+      </OverlayTrigger>
+      <OverlayTrigger
+        overlay={props.availableTooltipFunction(props.max, props.now)}
+        placement="top"
+        trigger={['hover', 'focus']}
+        rootClose={false}
+      >
+        <ProgressBar
+          min={props.min}
+          max={props.max}
+          now={props.max - props.now}
+          key={2}
+          bsClass="progress-bar progress-bar-remaining"
+          isChild
+        />
+      </OverlayTrigger>
+    </div>
+  );
+};
+
+UtilizationBar.propTypes = {
+  min: PropTypes.number,
```

Comment:
to add support for react-docgen in the "Show Info" tab (top right in Storybook), we have been adding prop comments like this:

```
/** progress min */
min: PropTypes.number
```